packages/marketing/src/nps/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormEvent } from 'react';
+
+function NpsScore( score: number ) {
+	return (
+		<label>
+			<input type="radio" name="nps-score" value={ score } />
+			{ score }
+		</label>
+	);
+}
+
+export function Nps() {
+	const scores = [ 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
+
+	const handleSubmit = ( event: FormEvent< HTMLFormElement > ) => {
+		event.preventDefault();
+	};
+
+	return (
+		<form onSubmit={ handleSubmit }>
+			<h1>How are we doing so far?</h1>
+			<fieldset>
+				<legend>
+					On a scale from 0–10, how likely are you to recommend WordPress.com to a friend or
+					colleague?
+				</legend>
+				<div className="nps-scale">
```

Comment:
For classnames we should probably follow some convention to avoid clashes. Something like `[name of package]-[name of component]__[...]`. So for this, maybe `marketting-nps__scale`?

Which brings up the question whether `marketting` might be too generic (or not)